src/Stryker.Core/Stryker.Core/Initialisation/InputFileResolver.cs

```diff
@@ -121,13 +141,17 @@ private FolderComposite FindInputFiles(string path, string projectUnderTestDir,
             return folderComposite;
         }
 
-        public string FindProjectFile(string basePath, string testProjectNameFilter)
+        public string FindProjectFile(string path)
```

Comment:
It seems I was a little too enthusiastic with removing the testProjectNameFilter here. It is still needed in the old situation. Maybe we can add a overload for the new situation and keep the old one as it is.